contracts/utils/InterchainAddressTracker.sol

```diff
@@ -29,14 +26,7 @@ contract InterchainAddressTracker is IInterchainAddressTracker, Upgradable {
         StringStorage.set(_CHAIN_NAME_SLOT, chainName_);
     }
 
-    /**
-     * @notice Getter for the contract id.
-     */
-    function contractId() external pure returns (bytes32) {
-        return CONTRACT_ID;
-    }
-
-    function _setup(bytes calldata params) internal override {
+    function _setupTrustedAddresses(bytes calldata params) internal {
```

Comment:
Hmm, this should be handled by the inheriting contract, in the constructor?

Reply:
Oops meant to delete it